index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+
+    <header class="app-header">
+        <a href="#" class="logo">Narson</a>
+        <nav class="ambony">
+            <a href="#"><img src="media/Union.svg" alt="humburger" class="humburger"></a>
+            <ul class="menu">
+                <li class="menu-item"><a href="#">Portfolio</a></li>
+                <li class="menu-item"><a href="#">About</a></li>
+                <li class="menu-item"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <section class="header-ambony">
+        <div class="header-container">
+        <h1 class="narson">I'm Narson.
+            <br>Glad to see you!
+        </h1>
+        <p class="detail">
+            I'm a software developer! I can 
+            help you build a product , feature or website Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+        </p>
+        <a href="#" class="connection"><h4>LET'S CONNECT</h4></a>
```

Comment:
Please remove the h4  tag from here`<a href="#" class="connection"><h4>LET'S CONNECT</h4></a>` and change it to `<a href="#" class="connection"><h4>LET'S CONNECT</h4></a>` since its not a heading and semantic also its not matching the figma design.